conf/application.conf

```diff
@@ -257,6 +257,12 @@ microservice {
             host = localhost
             port = 9197
             base-path = "/gform-stubs"
+            http-headers {
+                "X-Transmitting-System" = "CMA"
+                "X-Originating-System" = "MDTP"
+                "correlationid" = "{UUID}"
```

Comment:
Shouldn't `correlationid` header be an our `envelopeId`?

What is the value of generating random UUID we won't be able to know where it comes from.

Reply:
This will be handle in new ticket